test/cluster/blue-green-deployment/deploy.td

```diff
@@ -0,0 +1,88 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+> CREATE SCHEMA prod_deploy
+> CREATE SOURCE prod_deploy.counter IN CLUSTER prod_deploy FROM LOAD GENERATOR counter (TICK INTERVAL '1s')
+> CREATE MATERIALIZED VIEW prod_deploy.counter_mv IN CLUSTER prod_deploy AS SELECT count(*), 'some new value' FROM prod_deploy.counter
+> CREATE DEFAULT INDEX IN CLUSTER prod_deploy ON prod_deploy.counter
+> CREATE DEFAULT INDEX IN CLUSTER prod_deploy ON prod_deploy.counter_mv
+> CREATE SOURCE prod_deploy.tpch
+  IN CLUSTER prod_deploy
+  FROM LOAD GENERATOR TPCH (SCALE FACTOR 1)
+  FOR ALL TABLES
+> CREATE MATERIALIZED VIEW prod_deploy.tpch_mv
+  IN CLUSTER prod_deploy AS
+  SELECT
```

Comment:
You may wish to have a dedicated constant column that distinguishes the original view from the swapped-in one. Then check that the value of that column is as expected post-swap.